mne/event.py

```diff
@@ -885,3 +886,324 @@ def concatenate_events(events, first_samps, last_samps):
+    events : list
+        List of events that are in use (referenced by some averaging category).
+    reject : dict
+        Rejection criteria from DACQ that can be used with mne.Epochs.
+        Note that mne does not support all DACQ rejection criteria
+        (e.g. spike, slope)
+    flat : dict
+        Flatness criteria from DACQ that can be used with mne.Epochs.
+    """
+
+    # averager related DACQ variable names (without preceding 'ERF')
+    _dacq_vars = ('magMax', 'magMin', 'magNoise', 'magSlope', 'magSpike',
+                  'megMax', 'megMin', 'megNoise', 'megSlope', 'megSpike',
+                  'eegMax', 'eegMin', 'eegNoise', 'eegSlope', 'eegSpike',
+                  'eogMax', 'ecgMax', 'ncateg', 'nevent', 'stimSource',
+                  'triggerMap', 'update', 'version', 'artefIgnore',
+                  'averUpdate')
+
+    _event_vars = ('Name', 'Channel', 'NewBits', 'OldBits', 'NewMask',
+                   'OldMask', 'Delay', 'Comment')
+
+    _cat_vars = ('Comment', 'Display', 'Start', 'State', 'End', 'Event',
+                 'Nave', 'ReqEvent', 'ReqWhen', 'ReqWithin',  'SubAve')
+
+    def __init__(self, info):
+        acq_pars = info['acq_pars']
+        if not acq_pars:
+            raise ValueError('No acquisition parameters')
+        self.acq_dict = _acqpars_dict(acq_pars)
+        if 'ERFversion' not in self.acq_dict:
+            raise ValueError('Cannot parse version. The file may be from '
+                             'DACQ <3.4 which is not supported yet')
+        # set instance variables
+        for var in ElektaAverager._dacq_vars:
+            val = self.acq_dict['ERF' + var]
+            if var[:3] in ['mag', 'meg', 'eeg', 'eog', 'ecg']:
+                val = float(val)
+            elif var in ['ncateg', 'nevent']:
+                val = int(val)
+            setattr(self, var.lower(), val)
+        self.stimsource = (
+            'Internal' if self.stimsource == '1' else 'External')
+        # collect all events and categories
+        self._events = self._events_from_acq_pars()
+        self._categories = self._categories_from_acq_pars()
+        # mark events that are used by a category
+        for cat in self._categories.values():
+            if cat['event']:
+                self._events[cat['event']]['in_use'] = True
+            if cat['reqevent']:
+                self._events[cat['reqevent']]['in_use'] = True
+        # make mne rejection dicts based on the averager parameters
+        self.reject = {'grad': self.megmax, 'mag': self.magmax,
+                       'eeg': self.eegmax, 'eog': self.eogmax,
+                       'ecg': self.ecgmax}
+        self.reject = {k: float(v) for k, v in self.reject.iteritems()
+                       if float(v) > 0}
+        self.flat = {'grad': self.megmin, 'mag': self.magmin,
+                     'eeg': self.eegmin}
+        self.flat = {k: float(v) for k, v in self.flat.iteritems()
```

Comment:
``iteritems`` is not supported by some versions of python. I think you can use ``items()`` instead.